### DISCUSSION TITLE
QDate Not working when disable only certain date

### DISCUSSION THREAD

**Discussion by lamseck-jonathan** [quasarframework/quasar#14790] [General - Components / Directives / etc] [answered]
Is it possible to disable certain date only ? 
like i have an app of reservation , and what i want to achieve is retrieve all reservation and disable the date in QDate 
Everything is avaible if it is not in this period of time (reservation)
The actual behaviour is to enter the date that are available and it is hard to list all the dates that are available because there are so many 

I arleady tryed the function 
return date <= beginReservation && date >= endReservation
but i always return false and the whole QDate is disabled

**Comment by yusufkandemir** [comment, marked as answer]
See https://quasar.dev/vue-components/date#limiting-options

> I arleady tryed the function
return date <= beginReservation && date >= endReservation
but i always return false and the whole QDate is disabled

Your logic might be wrong. Create and share a reproduction(_Codepen or StackBlitz_), then explain what's wrong and what should happen clearly, so we can try to help.

**Reply by lamseck-jonathan** [reply]
here is the codepen
the logic that i use is there
https://codepen.io/lamseck-jonathan/pen/ZERWMjQ

**Reply by yusufkandemir** [reply]
You do the following:
```js
date <= "2022/11/09" && date >= "2022/11/16"
```
which means the `date` must be **both** earlier than `2022/11/09` **and** later than `2022/11/16` which is impossible. So you have a logic error there. You need to change it to `||`(or).
```js
date <= "2022/11/09" || date >= "2022/11/16"
```

**Reply by lamseck-jonathan** [reply]
Yes , it is working now , it was effectively an error of logic , thank you
